flags_test.go

```diff
@@ -213,3 +218,41 @@ func (t *FlagsTest) Maps() {
 	ExpectEq("", f.MountOptions["rw"])
 	ExpectEq("jacobsa", f.MountOptions["user"])
 }
+
+func (t *FlagsTest) ResolvePathInParentProcess() {
```

Comment:
One unit tests should focus only on one scenario

ResolvePathWhenParentProcessDirIsNotSetAndFilePathStartsWithTilda